tests/unit/moduleapi/list.tcl

```diff
@@ -1,5 +1,13 @@
 set testmodule [file normalize tests/modules/list.so]
 
+proc verify_list_edit_reply {reply argv} {
+    if {[dict exists $argv "i"]} { assert_equal [dict get $reply "inserts"] [dict get $argv "i"]}
+    if {[dict exists $argv "d"]} { assert_equal [dict get $reply "deletes"] [dict get $argv "d"]}
+    if {[dict exists $argv "r"]} { assert_equal [dict get $reply "replaces"] [dict get $argv "r"]}
+    if {[dict exists $argv "index"]} { assert_equal [dict get $reply "index"] [dict get $argv "index"]}
+    if {[dict exists $argv "entry"]} { assert_equal [dict get $reply "entry"] [dict get $argv "entry"]}
+}
+
```

Comment:
OK. This looks good. Only one minor idea. Now it is almost possible to write like this:

```suggestion
proc verify_list_edit_reply {reply argv} {
    foreach {k v} $argv {
        assert_equal [dict get $reply $k] $v
    }
}
```

This would be possible if `list.edit` would return the names "i", "r", "d" here. I think it's ok to use these short names since they are already the input commands to `list.edit`.

Reply:
Good idea, thx.